include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -64,37 +64,23 @@
 #    define _DPSTD_DEPRECATED_MSG(msg)
 #endif
 
-// Check the user-defined macro for parallel policies
-#if defined(ONEDPL_USE_TBB_BACKEND)
-#    undef _ONEDPL_USE_PAR_POLICIES
-#    define _ONEDPL_USE_PAR_POLICIES ONEDPL_USE_TBB_BACKEND
-// Check the internal macro for parallel policies
-#elif !defined(_ONEDPL_USE_PAR_POLICIES)
-#    define _ONEDPL_USE_PAR_POLICIES 1
-#endif
+#define _ONEDPL_PAR_BACKEND_SERIAL 1
 
-#if defined(ONEDPL_USE_OPENMP_BACKEND)
-#    undef _ONEDPL_USE_PAR_POLICIES
-#    define _ONEDPL_USE_PAR_POLICIES ONEDPL_USE_OPENMP_BACKEND
-// Check the internal macro for parallel policies
-#elif !defined(_ONEDPL_USE_PAR_POLICIES)
-#    define _ONEDPL_USE_PAR_POLICIES 1
+#if ONEDPL_USE_TBB_BACKEND || (!defined(ONEDPL_USE_TBB_BACKEND) && !ONEDPL_USE_OPENMP_BACKEND)
```

Comment:
It may work for now, but in the long term I would like the definition of `_ONEDPL_PAR_BACKEND_TBB` not depend on OpenMP-related macros, as that would allow us to provide specific policies (or executors) for all backends that can be used interchangeably.

In other words, I prefer the logic of backend availability and the logic of backend choice to be completely separate.

Reply:
Yes, agree. This is the place in the patch that I don't like either. I also thought about separation but need to think how to do it right.